packages/fxa-auth-server/lib/routes/auth-schemes/refresh-token.js

```diff
@@ -55,11 +55,19 @@ module.exports = function schemeRefreshTokenScheme(config, db) {
           return h.unauthenticated(AppError.invalidScopes(refreshToken.scope));
         }
 
+        const { ua } = request.app;
```

Comment:
If app.ua is not provided. The following code will fail in an a nondescript way. I'd suggest either making ua not required by providing a fallback state, or throwing a better message, ie and AppError.  I sort of think we should use a fallback state since throwing an error would technically be a breaking change to the API.